File: BookStore.APILayer/ClientApp/src/app/models/user.model.ts
import { UserBookModel } from "./userbook.model";
import { BasketModel } from "./basket.model";

export class UserModel {
        public id: number;
        public email: string;
        public password: string;
        public role: string;
        public confirmed: boolean;
        public userBook: Array<UserBookModel>;
        public userBasket: Array<BasketModel>;
}

File: BookStore.BusinessLogicLayer/Models/UserModel.cs
using System.Collections.Generic;

namespace BookStore.BusinessLogicLayer.Models
{
    public class UserModel
    {
        public int ID { get; set; }
        public string Email { get; set; }
        public string Password { get; set; }
        public string Role { get; set; }
        public bool Confirmed { get; set; }
        public ICollection<UserBookModel> UserBook { get; set; }
        public ICollection<BasketModel> UserBasket { get; set; }
    }
}
